openAPI/core-back-external.yaml

```diff
@@ -23,19 +47,34 @@ paths:
 
   /user-identity:
     get:
-      description: "Returns a list of Verifiable Credentials representig the users identity"
+      description: "List of Verifiable Credentials and claims representing the user's identity"
+      security:
+        - bearerAuth: ["user-identity"]
```

Comment:
What does `"user-identity"` here signify?

Reply:
These are the OAuth scopes required...

This is a bit of a bodge tbh, since this only does anything when we use the 'OAuth' security scheme... If we do that, then it wants to know about the authorization/token endpoint etc. required, which is confusing when we're defining those explicitly ourselves. See: https://swagger.io/docs/specification/v3_0/authentication/oauth2/#scopes-extra

Could just remove it, since it doesn't really do anything.

Reply:
I'm not sure the anchor in your link works properly, and I couldn't see what you were referring to?
This was confusing to me because these endpoints are protected by bearer auth, but it's the access token. So having the scope in here doesn't seem to make sense.
I think if we don't need it, maybe remove it?